django/forms/forms.py

```diff
@@ -191,6 +200,9 @@ def add_initial_prefix(self, field_name):
 
     def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_on_separate_row):
         "Output HTML. Used by as_table(), as_ul(), as_p()."
+        warnings.warn(
+            '`django.forms.BaseForm._html_output is deprecated` is deprecated.'
```

Comment:
I believe there is a typo here.

```suggestion
            '`django.forms.BaseForm._html_output` is deprecated.'
```